Modal windows for Mobile and Desktop versions with JavaScript

sebalp100 opened the pull request:
### **Modal windows for Mobile and Desktop versions with JavaScript**

In this milestone we added:
- New modals windows for every project with the following feature: 
  1. When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  2. When the user clicks (or taps) the close (X) button, the popup disappears.
- The new sections are generated with JavaScript
- Added objects to store projects data

BenMukebo commented on index.html:
```diff
@@ -188,7 +188,7 @@ <h2>Multy-Post Stories</h2>
             </li>
           </ul>
-          <button class="project-button">See Project</button>
+          <button class="project-button projButtFour">See Project</button>
```
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 


- Please blur the sub background when the popup is open, it will make the popup more like Figma 👍🏿
- It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 
- Kindly remove the extra padding-top and bottom of your popup text tittle `Multy-Post Stories` 
- Make sure the max width of your popup windows image takes the full width as shown on the Figma design
- Let's add the missing icons to the  `See live and See source` button ✍🏻 

Your Implementation | Figma Design
:---:|:---:
![18](https://user-images.githubusercontent.com/79058364/195240439-35fbd2b5-d128-4cad-a0a2-029fe44a7945.png)|![Screenshot from 2022-10-12 05-00-43](https://user-images.githubusercontent.com/79058364/195239977-37fb4625-af99-43e0-8509-0edd240c0d50.png)
 


- [ ]  Please do the same for the desktop size 👨🏻‍🎨 🎨 

- Actually, I don't see the full height of your page popups when I'm on the desktop size, please make sure 100% height is shown 👨🏻‍🎨

Your Implementation | Figma Design
:---:|:---:
![18](https://user-images.githubusercontent.com/79058364/195241361-058717ce-7d62-4c4b-b52d-b7c422f488c5.png) | ![Screenshot from 2022-10-12 05-13-25](https://user-images.githubusercontent.com/79058364/195241407-f696f0f5-78d1-4853-b3ea-baeebab49b3d.png)

sebalp100 replied:
Hi, that's the mobile version in a bigger resolution. Since the small cards have display none on bigger resolutions (768+px) the design looks like this: 
![desktop version](https://user-images.githubusercontent.com/111214189/195247652-ed3fcbe8-2242-49dd-83c3-117db57310d4.png)

BenMukebo reviewed (changes requested):
Hi @Team 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- No linter errors. ✔️ 
- GitHub flow was correctly used.  ✔️   


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sebalp100 commented on index.html:
```diff
@@ -107,7 +107,7 @@ <h2>Tonic</h2>
             </li>
           </ul>
-          <button class="project-button">See Project</button>
+          <button class="project-button projButtOne">See Project</button>
```
Hi @BenMukebo. I'm a little confused about having to redo the "works" section because the info from the objects is different in the template. Do I need to remove that section from the HTML file and create new objects for the normal section of the cards in javascript?

BenMukebo replied:
Yeah

sebalp100 commented:
Ok, I made all the changes required:

- I made the "works" section from scratch using javascript
- Added dark background when the popup is open
- Removed the scroll from the background when the scroll is open
- Removed the extra padding from the h2
- The images now take max width when expanding the screen
- Popup buttons now have the icons
- Desktop version now looks like the template

sja-thedude reviewed (changes requested):
## Required Changes ♻️

Hi @sebalp100, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![200w](https://user-images.githubusercontent.com/70269507/159484303-3187dacc-4a82-4263-bb1a-8d72417ce1d9.gif)

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [x] Kindly follow [this](https://www.loom.com/share/17e312c19993473d8cdd717b263de654) loom video to make small changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sebalp100 commented:
Hi @sja-thedude , sorry I forgot about the line. So I fix it by adding it to the mobile version 👍  :
![line](https://user-images.githubusercontent.com/111214189/195467326-963aaa23-f3cb-4298-8c66-8591246a206c.png)

but for some reason I can't see the padding from the title you are mentioning, it shows up like this for me:
![nopadding](https://user-images.githubusercontent.com/111214189/195467497-101a5871-0d2d-4a07-8e74-0b54e5f9c9e7.png)

sja-thedude commented:
> Hi @sja-thedude , sorry I forgot about the line. So I fix it by adding it to the mobile version 👍  :
> 
> ![line](https://user-images.githubusercontent.com/111214189/195467326-963aaa23-f3cb-4298-8c66-8591246a206c.png)
> 
> 
> 
> but for some reason I can't see the padding from the title you are mentioning, it shows up like this for me:
> 
> ![nopadding](https://user-images.githubusercontent.com/111214189/195467497-101a5871-0d2d-4a07-8e74-0b54e5f9c9e7.png)
> 
> 

Okay so try to deploy your app and check if the padding is seen in the deployment? If no then maybe it was some error from my side. Then you can submit for review.

sebalp100 commented:
> > Hi @sja-thedude but for some reason I can't see the padding from the title you are mentioning, it shows up like this for me:
> > ![nopadding](https://user-images.githubusercontent.com/111214189/195467497-101a5871-0d2d-4a07-8e74-0b54e5f9c9e7.png)
> 
> Okay so try to deploy your app and check if the padding is seen in the deployment. If no then maybe it was some error from my side. Then you can submit for review.

@sja-thedude  I managed to fix it by removing an extra div that was being created by mistake 👍 , so now it looks like this on 75%: 
![finalpadding](https://user-images.githubusercontent.com/111214189/195471168-07ca411b-f0f0-412e-b2d3-c10905627d81.png)

BenMukebo reviewed (approved):
### Hi @sebalp100, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 

![](https://media.giphy.com/media/vmtxnxveVUodG/giphy.gif)

### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your design look fine 👍🏻 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
